src/devices/device_models/AC_branches.jl

```diff
@@ -51,9 +51,8 @@ end
 function branch_rate_constraint(ps_m::CanonicalModel,
```

Comment:
Branch_rate_constraint is the same for `DCPlossless` and `StandardPTDF` why not use a `Union` and reduce code repetition.